docs/source/app_developers_guide/adding_a_node.rst

```diff
@@ -0,0 +1,443 @@
+   node has the same genesis block and treats all other nodes as peers.
+
+
+Docker: Start a Multiple-node Sawtooth Network
+==============================================
+
+#. If the first validator node is running, stop the Sawtooth components and run
+   ``docker-compose down``, as described in :ref:`stop-sawtooth-docker-label`.
+
+#. Create a Sawtooth application development environment as described in
+   :doc:`docker`, with the following important differences:
+
+   a. Download the Docker Compose file for a multiple-node network,
+      `sawtooth-default-poet.yaml <./sawtooth-default-poet.yaml>`_. Save this
+      file in the same directory as the single-node compose file
+      (``sawtooth-default.yaml``).
+
+   #. Use the following command to start the multiple-node Sawtooth network:
+
+      .. code-block:: console
+
+         user@host$ docker-compose -f sawtooth-default-poet.yaml up
+
+   #. This Compose file creates five validator nodes, numbered from 0 to 4.
+      Note the container names for the Sawtooth components on each node:
+
+      ``validator-0``:
+
+        * ``sawtooth-poet-validator-0``
+        * ``sawtooth-poet-rest-api-0``
+        * ``sawtooth-poet-settings-tp-0``
+        * ``sawtooth-poet-intkey-tp-python-0``
+        * ``sawtooth-poet-xo-tp-python-0``
+        * ``sawtooth-poet-validator-registry-tp-0``
+
+      ``validator-1``:
+
+        * ``sawtooth-validator-default-1``
+        * ``sawtooth-poet-rest-api-1``
+        * ``sawtooth-settings-tp-default-1``
+        * ``sawtooth-poet-intkey-tp-python-1``
+        * ``sawtooth-poet-xo-tp-python-1``
+        * ``sawtooth-poet-validator-registry-tp-1``
+
+      ... and so on.
+
+   #. Note that there is only one shell container:
+
+      * ``sawtooth-poet-shell``
+
+#. Configure the transaction family settings on the first validator node.
+
+   a. Connect to the ``sawtooth-poet-validator-0`` container.
+
+      .. code-block:: console
+
+         user@host$ docker exec -it ``sawtooth-poet-validator-0`` bash
+         root@validator-0#
+
+   #. Run the following command to create and submit a batch containing the
+      new settings:
+
+      .. code-block:: console
+
+         root@validator-0# sawset proposal create --url http://rest-api:8008 \
+         --key /root/.sawtooth/keys/my_key.priv \
+         sawtooth.validator.transaction_families='[{"family": "intkey", "version": "1.0"},{"family":"sawtooth_settings", "version":"1.0"},{"family":"xo", "version":"1.0"},{"family":"sawtooth_validator_registry", "version":"1.0"}]'
+
+#. You can connect to a validator registry container, such as
+   ``sawtooth-poet-validator-registry-tp-0``, then use the following command
+   to verify that ``poet-validator-registry-tp`` is running.
+
+   .. code-block:: console
+
+      # ps --pid 1 fw
+
+#. If you need to stop or reset the environment, enter CTRL-c in each
+   container to stop the Sawtooth component, then run the following command
```

Comment:
I think you can just do `docker-compose -f sawtooth-default-poet.yaml down`.  I don't think the ctrl-c is necessary and in fact sometimes it may not work.
Someone feel free to correct me if they have a different experience.

Reply:
CTRL-c in each container is definitely not the way to go. ctrl-c-ing the docker-compose command will stop each container and `docker-compose down` will reset the environment.

Reply:
I usually do CTRL-C in the 'docker-compose up' window and then run docker-compose down after that, which cleans things up.